hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/TokenServiceImpl.java

```diff
@@ -134,57 +135,42 @@ public void migrate(@NonNull MigrationContext ctx) {
-        return StateDefinition.inMemory(PAYER_RECORDS_KEY, keySerdes, valueSerdes);
-    }
-
-    private StateDefinition<EntityNum, MerkleToken> tokensDef() {
-        final var keySerdes = new EntityNumCodec();
-        final var valueSerdes =
-                MonoMapCodecAdapter.codecForSelfSerializable(MerkleToken.CURRENT_VERSION, MerkleToken::new);
-        return StateDefinition.inMemory(TOKENS_KEY, keySerdes, valueSerdes);
-    }
-
-    private StateDefinition<EntityNumVirtualKey, OnDiskTokenRel> onDiskTokenRelsDef() {
-        final var keySerdes = MonoMapCodecAdapter.codecForVirtualKey(
-                EntityNumVirtualKey.CURRENT_VERSION, EntityNumVirtualKey::new, new EntityNumVirtualKeySerializer());
-        final var valueSerdes =
-                MonoMapCodecAdapter.codecForVirtualValue(OnDiskTokenRel.CURRENT_VERSION, OnDiskTokenRel::new);
-        return StateDefinition.onDisk(TOKEN_RELS_KEY, keySerdes, valueSerdes, MAX_TOKEN_RELS);
-    }
-
-    private StateDefinition<UniqueTokenKey, UniqueTokenValue> onDiskNftsDef() {
-        final var keySerdes = MonoMapCodecAdapter.codecForVirtualKey(
-                UniqueTokenKey.CURRENT_VERSION, UniqueTokenKey::new, new UniqueTokenKeySerializer());
-        final var valueSerdes =
-                MonoMapCodecAdapter.codecForVirtualValue(UniqueTokenValue.CURRENT_VERSION, UniqueTokenValue::new);
-        return StateDefinition.onDisk(NFTS_KEY, keySerdes, valueSerdes, MAX_MINTABLE_NFTS);
+    private void updateNetworkRewards(final MigrationContext ctx) {
+        // Set genesis network rewards state
+        final var networkRewardsState = ctx.newStates().getSingleton(STAKING_NETWORK_REWARDS_KEY);
+        final var networkRewards = NetworkStakingRewards.newBuilder()
+                .pendingRewards(0)
+                .totalStakedRewardStart(0)
+                .totalStakedStart(0)
+                .stakingRewardsActivated(true)
+                .build();
+        networkRewardsState.put(networkRewards);
     }
 
-    private StateDefinition<EntityNum, MerkleStakingInfo> stakingInfoDef() {
-        final var keySerdes = new EntityNumCodec();
-        final var valueSerdes =
-                MonoMapCodecAdapter.codecForSelfSerializable(MerkleStakingInfo.CURRENT_VERSION, MerkleStakingInfo::new);
-        return StateDefinition.inMemory(STAKING_INFO_KEY, keySerdes, valueSerdes);
+    private void addAccount(
+            final AccountID id,
+            final WritableKVState<Object, Object> accounts,
+            final Key superUserKey,
+            final long expiry) {
+        accounts.put(
+                id,
+                Account.newBuilder()
+                        .receiverSigRequired(false)
+                        .deleted(false)
+                        .memo("")
+                        .smartContract(false)
+                        .key(superUserKey)
+                        .autoRenewSecs(expiry) // TODO is this right?
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Shouldn't we set only `expiry()` instead of `autoRenewSecs()` if the method is to be used for 800 and 801 accounts?

Reply:
Agreed. Changed to be similar to mono-service. Thanks @MiroslavGatsanoga